Client/Application/AppDelegate.swift

```diff
@@ -285,3 +202,40 @@ extension AppDelegate {
         menuBuilderHelper?.mainMenu(for: builder)
     }
 }
+
+// MARK: - Scenes related methods
+extension AppDelegate {
+
+    /// UIKit is responsible for creating & vending Scene instances. This method is especially useful when there
+    /// are multiple scene configurations to choose from.  With this method, we can select a configuration
+    /// to create a new scene with dynamically (outside of what's in the pList).
+    func application(
+        _ application: UIApplication,
+        configurationForConnecting connectingSceneSession: UISceneSession,
+        options: UIScene.ConnectionOptions
+    ) -> UISceneConfiguration {
+
+        let configuration = UISceneConfiguration(
+            name: connectingSceneSession.configuration.name,
+            sessionRole: connectingSceneSession.role
+        )
+
+        configuration.sceneClass = connectingSceneSession.configuration.sceneClass
+        configuration.delegateClass = connectingSceneSession.configuration.delegateClass
+
+        return configuration
+    }
+
+    /// Invoked by the system when an open scene session is discarded. It can be discarded by a user interaction,
+    /// or by the system itself (in the background as well).
+    ///
+    /// Use this method to release resources specific to that discarded scene.
+    /// Clean anything that holds memory or can deadlock resources for other scenes.
+    func application(
+        _ application: UIApplication,
+        didDiscardSceneSessions sceneSessions: Set<UISceneSession>
+    ) {
+        // no-op
+    }
```

Comment:
@nishant2718 is this for any future work? You aren't using this method or are you?